thresher/models.py

```diff
@@ -7,7 +7,8 @@ class UserProfile(models.Model):
     user = models.OneToOneField(User)
 
     # All topics have a set of users associated with them, so add a link to the parent Topic
-    topic = models.ForeignKey(Topic, on_delete=models.CASCADE, related_name="users")
+    topic = models.ForeignKey("Topic", on_delete=models.CASCADE, related_name="users")
```

Comment:
I am not quite sure why Users would have a ForeignKey to a Topic? Is this intended to represent the list of Topics a User is allowed to answer questions about? If that's the case then this would be a Many to Many relationship since one user can answer multiple topics and one topic will have multiple users answering. You would want to use a ManyToManyField for that. For reference: https://docs.djangoproject.com/en/1.9/topics/db/examples/many_to_many/